includes/Settings.php

```diff
@@ -1,13 +1,15 @@
 <?php
 
+namespace File_Upload_Types;
+
 defined( 'ABSPATH' ) || exit;
 
 /**
  * File Upload Types Settings.
  *
  * @since 1.0.0
  */
-class File_Upload_Types_Settings {
+class Settings {
```

Comment:
Do not place the actions (`add_action()` and `add_filter()` calls) inside `__contruct()` method of classes. It's better to have a separate `init()` or `hooks()` method that will hold all of these calls. This way you can avoid initializing the class inside the class file, and you can do that in the plugin plugin file (Core.php or Plugin.php).
With that said I would write something like
```php
// Inside Core.php/Plugin.php and its method init():

add_action('admin_init', [ $this, 'register_admin_area' ] );
```

That method will have this code:
```php
public function register_admin_area() {

    $settings = new Settings();
	$settings->init();
}
```

Reply:
@slaFFik It seems `admin_init` is too late to initialize. I've changed to `init` instead.